cpp/cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -2371,30 +2371,30 @@ macro(build_grpc)
   set(ABSL_LIBRARIES)
 
   # Abseil libraries gRPC depends on
+  # Follows grpc++ package config template for link order of libraries
+  # https://github.com/grpc/grpc/blob/v1.35.0/CMakeLists.txt#L16361
   set(_ABSL_LIBS
```

Comment:
This is specific to the version of Abseil, right? 

Maybe there's some way to extract this order if we have the CMake and/or PkgConfig files available to us? That said, it still builds in CI where I think a few versions are used so it seems at least that's ok. But I'm worried this part is a little brittle.

Reply:
It's specific to the version of gRPC not Abseil.
gRPC may change using Abseil libraries.

I agree with your worry. I hope that we can improve this later.